.github/workflows/main.yml

```diff
@@ -0,0 +1,53 @@
+# This is a basic workflow to help you get started with Actions
+
+name: CI
+
+# Controls when the action will run. 
+on:
+  # Triggers the workflow on push or pull request events but only for the master branch
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  # This workflow contains a single job called "build"
+  build:
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+      - uses: actions/checkout@v2
+      
+      - name: Get current package version
+        id: package-version
+        uses: martinbeentjes/npm-get-version-action@v1.1.0
+
+      # Runs a single command using the runners shell
+      - name: Install dependencies and build
+        run: |
+          npm i
+          npm run build 
```

Comment:
nemalo by tu byť náhodou `build-all`?

Reply:
netusim, vybuildi to vsetko tak ako to ma, npm az tak nepoznam

Reply:
no len build robí len typescript.. tie commandy sú napísané v `package.json`. Build-all buildí toho omnoho viac

Reply:
zmenene na build-all